workspaces/adventure-pack/goodies/python3/src/traverse_preorder/__init__.py

```diff
@@ -0,0 +1,25 @@
+from collections import deque
```

Comment:
nit: I think this can be removed since we are using a LIFO stack

Reply:
Forgot to do it. Removed now.